src/bench/bench.h

```diff
@@ -48,6 +48,7 @@ struct Args {
     fs::path output_csv;
     fs::path output_json;
     std::string regex_filter;
+    bool one_iteration;
```

Comment:
nit, place/pack this just below the first bool member

Reply:
ok, sure, why not, sorting the fields by type makes as much sense as anything

Reply:
yes, they've been sorted by increasing size